sensing_launch/launch/aip_s1/pointcloud_preprocessor.launch.py

```diff
@@ -51,9 +51,9 @@ def generate_launch_description():
       remappings=[('/output', 'concatenated/pointcloud')],
       parameters=[
           {
-              'input_topics': '[/sensing/lidar/top/outlier_filtered/pointcloud, '
-                              '/sensing/lidar/left/outlier_filtered/pointcloud, '
-                              '/sensing/lidar/right/outlier_filtered/pointcloud]',
+              'input_topics': ['/sensing/lidar/top/outlier_filtered/pointcloud',
```

Comment:
@TakaHoribe I was testing the pointcloud_preprocessor with this command

    colcon build --packages-up-to sensing_launch && ros2 launch sensing_launch sensing.launch.xml sensor_model:=aip_s1

I remain with this error

```
Found class: rclcpp_components::NodeFactoryTemplate<pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent>
[component_container-1] [INFO] [1606768022.137686544] [sensing.lidar.pointcloud_preprocessor.pointcloud_preprocessor_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent>
[component_container-1] [INFO] [1606768022.141397556] [concatenate_data]: Subscribing to 3 user given topics as inputs:
[component_container-1] [INFO] [1606768022.141457058] [concatenate_data]:  - /sensing/lidar/top/outlier_filtered/pointcloud
[component_container-1] [INFO] [1606768022.141466537] [concatenate_data]:  - /sensing/lidar/left/outlier_filtered/pointcloud
[component_container-1] [INFO] [1606768022.141473772] [concatenate_data]:  - /sensing/lidar/right/outlier_filtered/pointcloud
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/concatenate_data' in container '/sensing/lidar/pointcloud_preprocessor/pointcloud_preprocessor_container'
[component_container-1] [WARN] [1606752560.413682191] [concatenate_data]: Skipped /sensing/lidar/left/outlier_filtered/pointcloud,/sensing/lidar/right/outlier_filtered/pointcloud,/sensing/lidar/top/outlier_filtered/pointcloud. Please confirm topic.(not_subscribed_topic_name size = )143
[component_container-1] terminate called after throwing an instance of 'rclcpp::exceptions::RCLInvalidArgument'
```

It looks like the `publish()` method in https://github.com/tier4/Pilot.Auto/blob/ros2/sensing/preprocessor/pointcloud/pointcloud_preprocessor/src/concatenate_data/concatenate_data_nodelet.cpp#L230 is called. I thought that since I don't have lidars connected, there should be no messages published. But it seems to be time-triggered and fails when there is nothing to concatenate and thus an expected failure on my laptop? Can you confirm this?